tf2onnx/graph_builder.py

```diff
@@ -55,8 +55,7 @@ def make_slice(self, kwargs, name=None, shapes=None, dtypes=None, return_node=Fa
             inputs = [data, starts, ends, axes, steps]
 
         # pro-process inputs and attr
-        if kwargs:
-            logger.warning("kwargs contains un-used key")
+        utils.make_sure(not kwargs, "kwargs contains un-used key")
```

Comment:
If the call uses the wrong kwargs, there is a problem with the tf2onnx code.  This should be an error.